src/OrchardCore.Cms.Web/OrchardCore.Cms.Web.csproj

```diff
@@ -11,7 +11,13 @@
     <WarningsNotAsErrors>612,618</WarningsNotAsErrors>
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
     <RazorRuntimeCompilation>false</RazorRuntimeCompilation>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings Condition="'$(RazorRuntimeCompilation)' == 'false'">enable</ImplicitUsings>
   </PropertyGroup>
+  <!-- For Unit Tests-->
+  <ItemGroup>
```

Comment:
Can we add it in `AssemblyInfo.cs` instead?

Reply:
Updated as per https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-6.0#basic-tests-with-the-default-webapplicationfactory